FF1Lib/Spellcrafter.cs

```diff
@@ -316,7 +316,7 @@ enum OOBSpellEffects
 							validroutines.Add(0x0E);
 						else
 						{
-							if (SpellTier(lockspell) - SpellTier(index) > -2 || SpellTier(lockspell) - SpellTier(index) < 2)
+							if (SpellTier(lockspell) - SpellTier(index) > 2 || SpellTier(lockspell) - SpellTier(index) < -2)

```

Comment:
If you use Math.abs( ) you don't have to compare to both positive and negative 2 and you will avoid errors like this. 

```
If (Math.abs(SpellTier(lockspell) - SpellTier(index) > 2)) ...
```

You have a few examples in this file you can update.